lib/cob_index/indexer_config.rb

```diff
@@ -83,7 +83,19 @@
 
 # Creator/contributor fields
 to_field "creator_t", extract_marc_with_flank("245c:100abcdejlmnopqrtu:110abcdelmnopt:111acdejlnopt:700abcdejqu:710abcde:711acdej", trim_punctuation: true)
-to_field "creator_facet", extract_marc("100abcdq:110abcd:111ancdj:700abcdq:710abcd:711ancdj", trim_punctuation: true)
+to_field "creator_facet", extract_marc("100abcdq:110abcd:111ancdj:700abcdq:710abcd:711ancdj", trim_punctuation: true), filter_stop_words(
+  [ "Alexander Street Press",
```

Comment:
@dkinzer What do you think about passing a list into this line that is defined at the top of the file?  Similar to this in custom.rb: GENRE_STOP_WORDS = /CD-ROM|CD-ROMs|Compact discs|Databases)/i

Some of these lists have the potential to get long and makes it easier to read the file.

Reply:
Yeah we could do that.  The list is pretty un-intrusive at the moment so I think it's easier to jut plop it in there where it's being used vs having to scroll up to see what those are.

Reply:
@dkinzer But if this is a practice that we are going to do for the already existing genre work, that list is very long.  So to me it makes sense to condense things and keep the to_fields more readable.

Reply:
@sensei100 agreed.  For consistency's sake I propose that we update the genre example to use a filter macro as I have done with this solution.  I can create that PR as soon as this gets merged.

Reply:
@dkinzer But with the length of that list, I would want that list to be formatted the way it is currently.  So, I would think this list should be formatted in the same way.

Reply:
@sensei100 as someone with dyslexia I can assure you that format is completely impenetrable. But sure when I create the PR for the genre solution I'll keep the list formatted that way.

Reply:
@dkinzer My understanding is that this is just a preliminary list of corporate names, so it is very likely to get longer. Perhaps we should check in with Holly to see what would be easiest for her to maintain and update in the future?

Reply:
@ebtoner, @sensei100 you both are arguing for a future state that we can be easily addressed in another iteration (when the need actually arises).  There is no need to add indirection in the current context.

Reply:
@dkinzer It is a matter of opinion on the indirection.  I find that the list makes it much harder to read the to_field definitions and is unnecessary.  In my opinion, the actual values of what is being excluded is not as important as the readability of the rest of the code.

Reply:
@sensei100 @dkinzer Let's chat about this at our standup tomorrow. I think that it would more productive to talk together about this over Zoom rather than through github comments at this point.

Reply:
Per conversation with @htomren ... this can be merged as is.  I'll create and submit a PR to move genre and company_name values to files where she will have a smoother time maintaining them.